public/index.html

```diff
@@ -45,6 +45,60 @@ <h3 class="oxford location-heading">Oxford</h3>
             </div>
           </div>
 
+          <div class="job">
+            <p><a class="job-title">Paid Web Developer Internship</a></p>
+            <div class="job-description">
+              <p>
+                We are excited to offer a paid 3-month, web developer internship at the Zooniverse.
+                The candidate would ideally be able to travel to office in Oxford (UK) but we won't
+                rule out a remote candidate. The internship will run over summer 2017 with the option
+                for flexible start and end dates.
+              </p>
+              <p>
+                As an intern you will work on a variety of projects, according to our current needs
+                as well as the interests and skills of the successful candidate.
+
+                Projects will involve a variety of work including but not limited to:
+                <ul>
+                  <li>Adding new front-end features to our site</li>
+                  <li>Working on new backend API services</li>
+                  <li>Building data visualization tools</li>
+                  <li>Extracting services from our API</li>
+                  <li>Operations work with Docker and AWS</li>
+                </ul>
+              </p>
+              <p>
+                You should apply if you have a desire to be a web developer and you have experience
+                with any of the following:
```

Comment:
Perhaps something along the lines of  'you would like to gain experience with any of the following:'